x-pack/plugins/observability_solution/dataset_quality/public/components/dataset_quality/table/columns.tsx

```diff
@@ -162,73 +154,49 @@ export const getDatasetQualityTableColumns = ({
   fieldFormats,
   canUserMonitorDataset,
   canUserMonitorAnyDataStream,
-  selectedDataset,
-  openFlyout,
   loadingDataStreamStats,
   loadingDegradedStats,
   showFullDatasetNames,
   isSizeStatsAvailable,
   isActiveDataset,
   timeRange,
+  urlService,
```

Comment:
What is the use of this urlService here? is to redirect to the details?

Reply:
Yes, urlService is required to create the locator

https://github.com/achyutjhunjhunwala/kibana/blob/6a6806bcb0107aef1095096929c1dfc60b638a44/x-pack/plugins/observability_solution/dataset_quality/public/components/dataset_quality/table/dataset_quality_details_link.tsx#L27-L29